src/main/java/org/ukma/spring/crooodle/service/impl/RoomServiceImpl.java

```diff
@@ -13,14 +17,24 @@ public Room getRoom(long id) {
         return null;
     }
 
+    public LoadRoomResponseDto loadRoom(long roomId) {
+        return null;
+    }
+
     @Override
-    public void updateRoom(Room room) {
+    public RoomCrudResponseDto updateRoom(@Valid RoomCrudRequestDto requestDto) {
+
         //TODO: implement necessary logic
+
+        return new RoomCrudResponseDto();
     }
 
     @Override
-    public void deleteRoom(long roomId) {
+    public RoomCrudResponseDto deleteRoom(long roomId) {
```

Comment:
Delete doesn't return anything. Just throw exception or error (for example in case room with given id doesn't exist) and client will handle error code. Otherwise client will get 200 OK